ui_test/POM_hiway_asg/pages/constant_var.py

```diff
@@ -0,0 +1,29 @@
+webdriver_path = '/home/hasher/HU_Files/week4/ui_test/chromedriver_linux64/chromedriver'
+testing_website_url = 'https://qa.hiway.hashedin.com/'
+orange_color = 'rgba(255, 87, 34, 1)'
+blue_color = 'rgba(63, 81, 181, 1)'
+pink_color = 'rgba(255, 64, 129, 1)'
+eight_hrs_thirty_min = 1110
```

Comment:
constant names should convey how they should be used. eg. this could be sample timesheet duration